### ISSUE TITLE
A manual is missing

### ISSUE THREAD

**Issue by thomasroefer** [brotherbard/gitx#55]
The manual at http://gitx.frim.nl/ is rather out-dated and not complete. For instance, the effect of dragging around branch labels is not explained. I would find it especially helpful to know which git commands with which parameters are executed as a result of the GUI commands. Some commands result in other things than one would expect, e.g., in detached heads.

**Comment by thomasroefer** [comment]
Let me just say that the gitx version maintained at this page is my favorite git tool. No other tool has such a compact interface without clutter. So keep up the excellent work!

**Comment by heipei** [comment]
Yeah, we know that. Also, displaying the git-commands executed is not an option since we don't use the porcelain methods of git, so displaying what is executed would in no way benefit you.

**Comment by brotherbard** [comment]
I think putting a log of the commands in the GUI is a bad idea. But for those who want to debug GitX there is a user default that will log the commands to the console.

<pre>defaults write nl.frim.GitX "Show Debug Messages" -bool yes</pre>


Replace the 'yes' with 'no' to turn off. 

Note that there are a lot of messages flying around and that this will slow GitX down noticeably.
